Mathlib/GroupTheory/Index.lean

```diff
@@ -42,16 +42,16 @@ open Cardinal
 
 variable {G G' : Type*} [Group G] [Group G'] (H K L : Subgroup G)
 
-/-- The index of a subgroup as a natural number, and returns 0 if the index is infinite. -/
+/-- The index of a subgroup as a natural number. Returns `0` if the index is infinite. -/
 @[to_additive "The index of a subgroup as a natural number,
 and returns 0 if the index is infinite."]
```

Comment:
```suggestion
 @[to_additive "The index of a subgroup as a natural number.
Returns 0 if the index is infinite."]
```
I'm not sure that the suggestion will work, as it is outside of the modified lines.

Reply:
I applied it manually.